src/storage/client.py

```
@@ -1924,3 +1930,228 @@ def get_concepts_for_table(self, table_id: str) -> list[dict[str, Any]]:
+            # Link to chunk for chapter/section summaries
+            elif summary.level in ("chapter", "section"):
+                chunk_query = GetChunk(summary.source_id)
+                chunk_result = self._execute_query(chunk_query)
+                if chunk_result.success and chunk_result.data:
+                    chunk_node = chunk_result.data.get("node", chunk_result.data)
+                    chunk_internal_id = self._extract_node_id(chunk_result.data) or (chunk_node.get("id") if chunk_node else None)
+                    if chunk_internal_id:
+                        link_query = LinkChunkSummary(chunk_internal_id, summary_internal_id)
+                        self._client.query(link_query)
+
+            return summary_internal_id
```

Ruff (0.14.10)

Consider moving this statement to an `else` block

(TRY300)